### DISCUSSION TITLE
Use an index found in a array as a selector to another array

### DISCUSSION THREAD

**Discussion by wolicodes** [grafana/grafana-json-datasource#188] [Q&A] [answered]
Considering this JSON
```
{
  "titles": ["foo", "bar"]
  "values": ["fooValue", "barValue"]
}
```
How would you path to barValue using the index of "bar" in the titles array?

**Comment by marcusolsson** [comment, marked as answer]
I'd have to know more about the use case, but could you use something like this?

```js
$.values[?(@ == @parent.titles[1]+"Value")]
```

<img width="674" alt="Screenshot 2021-10-13 at 15 30 56@2x" src="https://user-images.githubusercontent.com/8396880/137142532-c42f1191-e1e8-4e30-8f9c-bab6a0d8b8de.png">
